sql-cli/sql_cli/dag_generator.py

```diff
@@ -6,8 +6,8 @@
 
 from networkx import DiGraph, depth_first_search, find_cycle, is_directed_acyclic_graph
 
+from astro.utils.sql_directory_parser import SqlFile, get_sql_files
```

Comment:
@dimberman why are we placing it inside `utils`? I have the feeling it would be better to have it alongside the other tasks operators, WDYT?

Reply:
@tatiana but it's not an operator? Maybe have a "render" directory?

Reply:
Yes, `render` works for me. I think it is not fair to call this `utils`. It deserves a better name.